scipy/stats/unuran/unuran_wrapper.pyi

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from typing import TYPE_CHECKING, Union, Any, Callable, Tuple, Optional, List
+
+if TYPE_CHECKING:
+    import numpy.typing as npt
+
+
+SeedType = Union[np.random.RandomState, np.random.Generator, int,
+                 npt.ArrayLike]
+
+
+__all__: List[str]
+
+
+class Method:
+    ...
```

Comment:
Is this a class without any (public) methods or is this simply a work in progress?
In the latter case you should add a dummy `__getattr__` method so that users won't be saddled with loads of false positives due to missing attributes:
``` python
class Method:
    def __getattr__(self, name: str) -> Any: ...
```

Reply:
Or more ideally, just add the bare unannotated functions signatures, since adding `__getattr__` will make every single `getattr` operation legal (it's better than false positives, but that's still a pretty massive false negative).

Reply:
> Is this a class without any (public) methods

Actually, this is a base class with only `rvs` as a public method. But the class itself isn't public. I just wanted to let Mypy know that the classes `TransformedDensityRejection` and `DiscreteAliasUrn` use this as a base class. If it is not required, I can remove this altogether.